modules/main-rules/pages/transportation.adoc

```diff
@@ -2,12 +2,42 @@
 
 On some unit data sheets you'll see that the unit has one of two special abilities that relate to transportation: {transport-x} or {hitch-x}.
 We explain these and other special abilities in more detail elsewhere; but transportation is a vital aspect of the Movement phase so we'll discuss it here too.
+// IJW editing note - the second sentence isn't really true, this page is where  the detail is!
 
 When we speak of these transportation abilities, we use the terms 'embark/embarked' and 'disembark/disembarked' to describe actions and states that relate to the units that you transport.
 
 Both the {transport-x} and {hitch-x} abilities indicate units that can carry -- transport -- other units across the battlefield.
 But they work in slightly different ways.
 
+== Placement of units
+To embark a unit, it must be completely within 5 cm of the transportation unit.
+
+When you disembark a unit, place it completely within 5 cm of the transportation unit.
+
+[NOTE.e40k]
+.Placement of units
+====
+We added this rule to minimise the effects that result from the assortment of different base sizes that players use.
+====
+
+== Destruction of transportation units
+
+If a transportation unit is destroyed then, just before you remove the unit as a casualty, roll a D6 for each embarked unit: On a result of 1, 2 or 3, the unit suffers a hit.
+If the unit survives, place it completely within 5 cm of the destroyed transportation unit.
+
+Then remove the transportation unit as normal.
```

Comment:
Am I right to think that this situation doesn't actually arise with Hitch (X) units, except in the rare case of a failed Dangerous Terrain test followed by another roll of 1...?

If so then I think we should clarify this. And, this would strengthen the argument for us to move the Transport(X)-specific and Hitch(X)-specific sections over to the respective topics, given how little 'common' content that would leave us with for this topic.

In other words, it might be better to restore the 'Destruction of {transport-x} units' section or similar, and match it with a 'Destruction of {hitch-x} units' section that clearly shows this difference.

Reply:
Yes, Dangerous terrain tests, but if you're using lots of Hitch units and terrain it comes up every other game or so. However what's more common if you're using games with hidden setup is booby traps (hit on a 2+) and minefields (roll to hit every moving unit).

But that makes no difference to how the 'destruction of x' rule works, it's identical for both types of transportation unit. Maybe just a 'tip' note pointing out when Hitch units can be destroyed while carrying someone? Or an edition note saying that E40k didn't say what to do when a Hitch unit was destroyed while carrying troops.

Reply:
Okay, that's clear. The reason that I suggest a possible split here is that something like these two similar but different passages:

> If a Transportation (X) is destroyed then, just before you remove the unit as a casualty, roll a D6 for each embarked unit: On a result of 1, 2 or 3, the unit suffers a hit.

> If a Hitch (X) unit is destroyed while it is motion then, just before you remove the unit as a casualty, roll a D6 for each embarked unit: On a result of 1, 2 or 3, the unit suffers a hit.

...may turn out to be neater than a single passage in which we try to spell out the differences, with or without a note. Especially if we do find it better to move the ability-specific stuff to the ability topics and leave only the most generic stuff in the transportation topic.

I'm not sure though. I'll have a play with it soon, see how it looks.

Reply:
I think I'm missing something. What's the purpose of having two different passages? If a transportation unit is destroyed with units embarked on it, the game mechanics are identical.

'while it is in motion' definitely can't be used instead of a reminder of how the units can be destroyed, because everything that can kill a moving Hitch unit can also kill a moving Transport unit.

Reply:
>'while it is in motion' definitely can't be used instead of a reminder of how the units can be destroyed, because everything that can kill a moving Hitch unit can also kill a moving Transport unit.

I think you're sort of looking at it from the opposite end to me.

As I understand it, units are only 'embarked' on a Hitch (X) unit while it is in motion. We embark them at some point during the Hitch (X) unit's move, and disembark them at some other point in its move (or sequence of moves) -- often at the end. If the Hitch (X) unit is destroyed when it is at rest then there are no embarked units that might get hit -- which is different to a Transport (X) unit which may contain embarked units while it is at rest.

Hence why the 'in motion' bit might work as an elegant way to differentiate, while the alternative might be more clunky. The rest of the passage to determine hits works the same, sure, but the conditions to trigger this differ because they are more limited for Hitch (X).

Reply:
> Hence why the 'in motion' bit might work as an elegant way to differentiate, while the alternative might be more clunky. The rest of the passage to determine hits works the same, sure, but the conditions to trigger this differ because they are more limited for Hitch (X).

My issue is that including the 'in motion' bit for Hitch units infers that Transport units that die 'in motion' _don't_ follow these rules. Dying while in motion is a condition that applies to both types of transportation unit, so it can't be used as a differentiator, only as a reminder.

I don't think it's necessary to spell out the ways in which a transportation unit can die, but the text could be tweaked to reinforce it only applying while carrying other units:

```suggestion
== Destruction of transportation units

If a transportation unit is carrying other units and is destroyed then, just before you remove the unit as a casualty, roll a D6 for each embarked unit: On a result of 1, 2 or 3, the unit suffers a hit.
If the unit survives, place it completely within 5 cm of the destroyed transportation unit.

Then remove the transportation unit as normal.
```

If you want to spell it out,  a reminder note could be added to the {hitch-x} section of the page, for example:

>TIP: As {hitch-x} units can only carry units while they are moving, the _Destruction of transportation units_ rule only applies if the {hitch-x} unit dies during a move, for example to a failed Dangerous terrain test, or being hit by a Booby-trap or Mine-field.

Reply:
>My issue is that including the 'in motion' bit for Hitch units infers that Transport units that die 'in motion' don't follow these rules

Why should it infer any such thing? The 'in motion' bit is only for the Hitch (X) passage. The equivalent Transport (X) passage doesn't have the 'in motion' condition, so by default it applies to all conditions. That is literally the point.

There is more than one way to skin a cat, and I haven't experimented with it yet to see how it actually looks — but the point here is that those few words around 'in motion' might be more elegant than a big 'note' paragraph like the one you suggest. Unless you think that it is too subtle and would still need further explanation. But that's a different issue to inferring the wrong thing, which I can't see that it does, as above.

Reply:
> Why should it infer any such thing? The 'in motion' bit is only for the Hitch (X) passage. The equivalent Transport (X) passage doesn't have the 'in motion' condition, so by default it applies to all conditions. That is literally the point.

Hmm, let's start again.

Being destroyed is a trigger. Being in motion is not part of that trigger, but for Hitch units it's a prerequisite to _reach_ the trigger of being destroyed.

The text conflates the trigger and the prerequisite for that trigger, which means that the reader can't take the text at face value. They have to analyse the text to work out why the concepts are being conflated. And that can result in reading the passages like this:

If X is true, apply Z to Transport units.
In the separate situation where X _and_ Y are true, apply Z to Hitch units instead.

I think we can agree that we don't want the reader to even _suspect_ that this is the game logic!

----

Bear in mind that I think we're going to need an edition change note anyway, as destruction of Hitch units while carrying other units is missing in the original rules. If you want to spell out that Hitch units can only carry units while moving, so embarked units can only die during the move, and we need a note anyway, why not mention it in the note?

----

As an aside, and I know this is encroaching on your territory, is it a good idea to have all the transportation stuff split across three pages? I'm a bit concerned that the original rules forced the reader to flick back and forth in the book to work out how Transport (x) worked, and we're currently doing the same.

Might it be worth pulling the small amount of info from hitch-x.adoc and transport-x.adoc into transportation.adoc and leaving them as stubs, or expanding hitch-x.adoc and transport-x.adoc into complete entries with a partial for the parts that are the same for both, with transportation.adoc becomign a stub?

Reply:
I'm still not seeing whatever it is that you're seeing. But it was only an idea I wanted to check with you— I'll drop this specific iteration of it, as it seems to be failing that check.

---

>Bear in mind that I think we're going to need an edition change note anyway

Yes, but that means **two** notes, one of which we need no matter which way we do it — one for the actual clarification/change itself, and one to highlight the fact that it is a change from E40k.

>why not mention it in the note?

I don't believe it is a good idea to conflate the rule or clarification — that we would normally put in the main text or in a regular note — with the purpose of the 'E40k difference' note. The latter highlights and explains the former; it doesn't take its place.

---
>is it a good idea to have all the transportation stuff split across three pages?

It depends on how it is split, as you've alluded to yourself. There's more than one possible approach, and I haven't got a clear idea of which is likely to work best from just my mind's eye.

>the original rules forced the reader to flick back and forth in the book to work out how Transport (x) worked, and we're currently doing the same.

I'd say we're not, currently. The Hitch (X) and Transport (X) ability topics barely say anything at all — the real content is in the Transportation topic. However, I am wondering if it might work better to move the content to the ability topics, when there is so little that is truly in common anyway. I recall that you also pondered this very idea previously — I just figured that now might be the time to give it full consideration, as an alternative to deleting the line that you commented to the effect that it 'isn't really true' because it says that the detail is in the ability topics when in fact it isn't at present.

As I said, I need to experiment with it a bit I think. But I suspect that `stub Transportation topic + two short detailed ability topics` will work better than `1 long detailed Transportation topic + two stub ability topics`. For me, your confirmation that the conditions are indeed different reinforces this suspicion, because it leaves even less that is truly in common and without caveats between the two modes of transportation.

Reply:
> > Bear in mind that I think we're going to need an edition change note anyway
> 
> Yes, but that means **two** notes, one of which we need no matter which way we do it — one for the actual clarification/change itself, and one to highlight the fact that it is a change from E40k.
> 
> > why not mention it in the note?
> 
> I don't believe it is a good idea to conflate the rule or clarification — that we would normally put in the main text or in a regular note — with the purpose of the 'E40k difference' note. The latter highlights and explains the former; it doesn't take its place.

Can we start this discussion from scratch, from first principles?

- A Hitch unit moves, embarking and then **always disembarking** carried units during the same move/march.
- If the Hitch unit gets shot, or hit in close combat etc, it's impossible for it to be carrying other units at that time.
- Players never need to look up what happens if a stationary Hitch unit is destroyed while carrying units, because that situation cannot occur during a game.

If the game state can never include a situation, we don't need additional rules text to exclude that situation. 

As far as I can see, this means that all we need is an edition change note, saying that we've expanded the rules to cover Hitch units as they can die while carrying units.

Reply:
>* Players never need to look up what happens if a stationary Hitch unit is destroyed while carrying units, because that situation cannot occur during a game.

Um, yes... good point. While I recognised the difference, I think I had some sort of idea that players who are used to Transport (X) might fall foul of 'muscle memory' and apply the same conditions to Hitch (X) -- forgetting that most of the time hitching units aren't actually embarked (only actually embarked while 'in motion'). I still think that could happen quite easily; but now I take your point that it warrants only a clarifying note at most.

Okay, good -- on the same page at last, I was just a bit slow/side-tracked on this one! :-D

Reply:
I've pushed a bunch of edits, but there is more to do. Eg:

1. Some annotations to resolve.
2. It may yet be best to integrate the embark/disembark placement conditions into each set of transportation rules, rather than have them under a separate heading — I find it a bit disjointed.

Also note that, aside from rationalisation of text that defines what units each ability can carry, I have not split the ability-specific stuff off to the separate ability topics. I have misgivings about both approaches; but so far my experiments lead me to think that the current approach is more user-friendly, on balance. This does deviate from the approach we've taken with the likes of Artillery and Flak — but it is a bit of a special case.

I welcome your thoughts on this. Though of course we can always change it later if we have a brainwave, so no point in tying ourselves up in knots if the current approach is 'good enough'.

Reply:
Resolving this conversation as we reached a conclusion and it is now outdated.